web/src/pages/apd/proposed-budget/BudgetSummary.js

```diff
@@ -536,17 +538,21 @@ const BudgetSummary = ({ activities, data, years, apdType }) => {
           </div>
 
           <h3>Medicaid Total Computable by FFY</h3>
-          <p>
-            The Activities Total compiles all the summary budget tables by FFY
-            and funding source.
-          </p>
-          <Alert>
-            The Activities Grand Total is{' '}
-            <span className="ds-u-font-weight--bold">
-              <Dollars>{data.combined.total.medicaid}</Dollars>
-            </span>
-            .
-          </Alert>
+          {!isViewOnly ? (
```

Comment:
This works, although you don't need the `: null` option if you change it to: `!isViewOnly && <p>Stuff...</p>`